packages/core/src/icon/utils/icon.classnames.ts

```diff
@@ -131,20 +91,15 @@ export function getUpdateSizeStrategy(size: string) {
 
 export function updateIconSizeStyleOrClassnames(el: CdsIcon, size: string) {
```

Comment:
@mathisscott The name of the function can be renamed. I didn't do it because I am not sure if someone out there is not using it for some reason.

Reply:
These internal utility functions aren't documented so there is low risk that someone outside of the Clarity project is using them.